serializers/vm.rb

```diff
@@ -27,6 +27,10 @@ def self.serialize_internal(vm, options = {})
       )
     end
 
+    if options[:load_balancer]
+      base[:state] = DB[:load_balancers_vms].where(vm_id: vm.id).get(:state)
```

Comment:
It's not `vm[:state]`. We have already `state` property. I think `load_balancer_state` is more accurate for this case